README.rst

```diff
@@ -1,42 +1,42 @@
-OSGeo live documentation
-========================
-OSGeoLive_ is a self-contained bootable DVD, USB thumb drive or Virtual
```

Comment:
Why did you remove the description of OSGeoLive from the README?
Its the first thing people will read, it tells people what the repository is about